src/SmartComponents/BaselinesTable/redux/__tests__/baselinesTableReducer.fixtures.js

```diff
@@ -92,15 +92,15 @@ function baselineTableDataOneSelected() {
     return baselineTableDataOneSelected;
 }
 
-const tableDataCSV = 'UUID,Name,Last updated,Associated systems\n\
-1234,beavs baseline,4 years ago,3,true\n\
-abcd,micjohns baseline,4 years ago,0,false\n\
+const tableDataCSV = 'UUID,Name,Last updated,Associated systems,Notifications enabled\n\
+1234,beavs baseline,3 years ago,3,true\n\
```

Comment:
We should do something about these tests here. This is now the third PR or so where it pops up and is unrelated to the PR itself. We need to look into mocking time here and make sure that it'll always be 3 or 4 years, or both, but it should never change over time.

Reply:
those are now fixed here: https://github.com/RedHatInsights/drift-frontend/pull/616